frontend/sign-up.html

```diff
@@ -1,54 +1,57 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <title>Quadcore: Technology News Aggregator</title>
-        <!--Import Google Icon Font-->
-        <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-        <!--Import materialize.css-->
-        <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.5/css/materialize.min.css">
-        <!--Import custom css file-->
-        <link rel="stylesheet" type="text/css" href="css/sign-up.css">
-        <!--Let browser know website is optimized for mobile-->
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    </head>
-    <body>
-        <div class="container ">
-            <div class="card-panel z-depth-4 center">
-                <h4 class="header2">Register</h4>
-                <div class="row">
-                    <form id="sign-up-form" class="col s12" method="post">
-                        <div class="row">
-                            <div class="input-field col s6">
-                                <input id="first_name" type="text">
-                                <label for="first_name">First Name</label>
-                            </div>
-                            <div class="input-field col s6">
-                                <input id="last_name" type="text">
-                                <label for="last_name">Last Name</label>
-                            </div>
+
+<head>
+    <title>Quadcore: Technology News Aggregator</title>
+    <!--Import Google Icon Font-->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <!--Import materialize.css-->
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.97.5/css/materialize.min.css">
+    <!--Import custom css file-->
+    <link rel="stylesheet" type="text/css" href="dist/css/sign-up.css">
+    <!--Let browser know website is optimized for mobile-->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+</head>
+
+<body>
```

Comment:
In other files, you used indentation with 2 spaces, but here it's 4. Can we be consistent?